content/en/users/getting-started/cli/_index.md

```diff
@@ -1,11 +1,444 @@
+(_$VIRTUAL_ENV/bin_ for executing pip3 in a Python virtual environment,
+_~/.local/bin_ for executing pip3 as user (with --user option), and
+_/usr/local/bin_ when executing pip3 as root).
+<!-- markdownlint-enable no-space-in-emphasis -->
+
+{{< /tabx >}}
+{{< tabx header="Windows" >}}
+
+As there are non-pure Python packages needed for installation, the
+[Microsoft C++ Build Tools](https://visualstudio.microsoft.com/visual-cpp-build-tools/)
+is a prerequisite, make sure it's installed with the following options
+selected:
+
+- C++ CMake tools for Windows
+- C++ ATL for latest v142 build tools (x86 & x64)
+- Testing tools core features - Build Tools
+- Windows 10 SDK (`<latest`>)
+
+In case you prefer to use non-Microsoft alternatives for building non-pure
+packages, please see [here](https://wiki.python.org/moin/WindowsCompilers).
+
+To install the FedCloud client:
+
+```shell
+> pip3 install fedcloudclient
+```
+
+This installs the latest version of the FedCloud client, together with
+its required packages (like _openstackclient_). It will also create
+executables **fedcloud** and **openstack**, adding them to the _bin_
+folder corresponding to your current Python execution environment.
+
+{{< /tabx >}}
+{{< /tabpanex >}}
+
+Check if the installation is correct by executing the client:
+
+```shell
+$ fedcloud --version
+```
+
+#### Installing EGI Core Trust Anchor certificates
+
+Some sites in the EGI infrastructure use certificates issued by
+Certificate Authorities (CAs) that are not included in the default OS
+distribution. If you receive error message "_SSL exception connecting
+to..._", install the EGI Core Trust Anchor Certificates by running
+the following commands:
+
+<!-- markdownlint-disable line-length -->
+```shell
+$ wget https://raw.githubusercontent.com/tdviet/python-requests-bundle-certs/main/scripts/install_certs.sh
+$ bash install_certs.sh
+```
+<!-- markdownlint-enable line-length -->
+
+{{% alert title="Note" color="info" %}} The above script does not work on all
+Linux distributions. Change _python_ to _python3_ in the script if needed,
+see the [README](https://github.com/tdviet/python-requests-bundle-certs#usage)
+for more details, or follow the
+[official instructions](https://github.com/tdviet/python-requests-bundle-certs/blob/main/docs/Install_certificates.md)
+for installing EGI Core Trust Anchor certificates in production environments.
+{{% /alert %}}
+
+### Using via Docker container
+
+The FedCloud client can also be used without installation, by running it in a
+Docker container. In this case, the EGI Core Trust Anchor certificates are
+preinstalled.
+
+{{< tabpanex >}}
+{{< tabx header="Linux " >}}
+
+To run the FedCloud client in a container, make sure
+[Docker is installed](https://docs.docker.com/engine/install/#server),
+then run the following commands:
+
+```shell
+$ sudo docker pull tdviet/fedcloudclient
+$ sudo docker run -it  tdviet/fedcloudclient bash
```

Comment:
```suggestion
$ sudo docker pull tdviet/fedcloudclient
$ sudo docker run -it tdviet/fedcloudclient bash
```

`sudo` may not be applicable, depending on the system is configured, may be we could just mention that it may be required to use sudo (or be in the proper system group) to use docker.